modules/admin_manual/pages/installation/apps_management_installation.adoc

```diff
@@ -5,105 +5,104 @@
+However, this is the exception.
+Apps are mainly configured from your ownCloud Personal or Admin settings page, or in `config.php`.
 
-After you add a new directory configuration, you can then move apps from
-the original app directory to the new one. To do so, follow these steps:
+== Using Custom App Directories
 
-1.  xref:configuration/server/occ_command.adoc#maintenance-commands[Enable maintenance mode].
-2.  xref:configuration/server/occ_command.adoc#apps-commands[Disable the apps] that you want to move.
-3.  Create a new apps directory and assign it the same user and group,
-and ownership permissions as the core apps directory.
-4.  Move the apps from the old apps directory to the new apps directory.
-5.  Add a new app directory in `config/config.php`.
-6.  If you’re using a cache, such as xref:configuration/server/caching_configuration.adoc#clearing-the-redis-cache[Redis] or xref:configuration/server/caching_configuration.adoc#clearing-the-memcached-cache[Memcached], ensure that you clear the cache.
-7.  Re-enable the apps.
-8.  Disable maintenance mode.
+There are several reasons for using custom app directories instead of ownCloud's default. 
+These are:
 
-== Manually Installing Apps
+. It separates ownCloud's core apps from user or admin downloaded apps. 
+Doing so distinguishes which apps are core and which aren't, simplifying upgrades.
+. It eases manual upgrades. 
+Having non-core apps in a directory separate to the core app directory makes them simpler to manage.
+. ownCloud may gain new core apps in newer versions. 
+Doing so orphans deprecated apps, but doesn't remove them.
 
-To install an app manually instead of by using https://marketplace.owncloud.com[the Marketplace], copy the app either into ownCloud’s default app folder (`</path/to/owncloud>/apps`) or xref:using-custom-app-directories[a custom app folder].
+If you want to store apps in a custom directory, instead of ownCloud's default (`/app`), you need to modify the `apps_paths` element in `config/config.php`. 
+There, you need to add a new associative array that contains three elements. 
+These are:
 
-Be aware that the name of the app and its folder name *must be
-identical*! You can find these details in
-the application's metadata file, located in
-`<app directory>/appinfo/info.xml`.
+`path`:: The absolute file system path to the custom app folder.
+`url`:: The request path to that folder relative to the ownCloud webroot, prefixed with `/`.
+`writable`:: Whether users can install apps in that folder. 
 
-Using the example below, both the app’s name and directory name would be
-`yourappname`.
+After adding the configuration, ownCloud only installs apps in directories where `writable` is set to `true`.
+The configuration example below shows how to add a second directory, called `apps-external`.
 
-[source,xml]
+[source,php]
 ----
-include::{examplesdir}installation/appinfo.xml[]
```

Comment:
Pls check if `appinfo.xml` is used somewhere else and in case not remove it to avoid orphans

Reply:
Thanks! Done.